sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ClipboardManager.kt

```diff
@@ -41,7 +42,7 @@ class RealClipboard @Inject constructor(
 
     override fun pasteFromClipboard(): String {
         return if (clipboardManager.hasPrimaryClip()) {
-            clipboardManager.primaryClip!!.getItemAt(0).text.toString()
+            clipboardManager.primaryClip?.getItemAt(0)?.text.takeUnless { it.isNullOrBlank() }?.toString() ?: ""
```

Comment:
Calling to String should not crash on a null object (`public fun Any?.toString(): String`). But just in case I added some defensive checks.

Reply:
That is the most defensive line of code i've ever seen 😂